astropy/units/tests/test_units.py

```diff
@@ -750,7 +739,7 @@ def test_compose_into_arbitrary_units():
     # Issue #1438
     from astropy.constants import G
 
-    G.decompose([u.kg, u.km, u.Unit("15 s")])
+    assert isinstance(G.decompose([u.kg, u.km, u.Unit("15 s")]), u.Quantity)
```

Comment:
My tendency would be to make this a more useful test. E.g.,
```
G_decompose = G.decompose([u.kg, u.km, u.Unit("100 s")])
assert_allclose(G_decompose.scale, 0.0001)
assert_allclose(abs(G_decompose - G).to_value(u.one), 0)
```

Reply:
I'm not opposed to editing the test, but your suggestion doesn't work because `G_decompose` is a `Quantity`, so it doesn't have a `scale` attribute.

Reply:
Sorry, yes, that was meant to be `G_decompose.unit.scale`.